cluster/addons/e2e-rbac-bindings/random-addon-grabbag.yaml

```diff
@@ -1,20 +0,0 @@
-# TODO remove this
-# currently, the kube-addon-manager is adding lots of pods which all share
-# the system:serviceaccount:kube-system:default identity.  We need to subdivide
-# those service accounts, figure out which ones we're going to make bootstrap roles for
-# and bind those particular roles in the addon yaml itself.  This just gets us started
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
tightening permissions on unknown workloads on upgrade isn't something we should do lightly. I'd take an ugly toothless binding for a release or two. Can release note it now, omit it in new clusters, and remove the stub in a couple releases.